cds-snc/covid-alert-portal
# Generate and send PDF poster

# Summary | Résumé

**Heads-up: see note below about new dependencies, you will probably need to install cairo before testing this PR**

Takes the SVG poster template and converts it to PDF then sends it as an attachment via Notify and makes it available for download on the Confirmation screen.

Note that this doesn't include the latest designs/content for some things:
- Old bilingual poster design (not en/fr)
- Old email content
- Confirmation screen link designs
- Content updates on Confirmation screen

## To test
- Step through the forms to submit information about your location
- On the Confirmation screen, click the "Download poster" link
- You should also have received an email with the poster attached

## To run on your local machine
- If you run the project with docker you should be good to go after a container rebuild - cairo dependency has been added to the Dockerfile
- If you run the project locally with pipenv, you will need to install cairo. On the mac: `brew install cairo`

## New dependencies
- Pipfile: cairosvg - for svg2pdf conversion
- Dockerfile: libcairo2-dev - os-level support for [Cairo](https://www.cairographics.org/download/)
